src/injected.ts

```diff
@@ -0,0 +1,75 @@
+import './encryption/aes';
+
+import { Communication } from './communication/injected/communication';
+import { StatusPayload, TYPE_ONOFF } from './communication/types';
+import * as aesHelper from './encryption/aes_helper';
+import Store from './encryption/store';
+import { goOverTeamsChatMessages } from './Teams/teams';
+
+let isActive = false;
+
+Store.addKey(aesHelper.STATIC_DEV_KEY_IV, "chatIdent");
+
+console.log("Trying to hook teams");
+interface EnhancedHTMLDivElement extends HTMLDivElement {
+  lastListenerInfo: Array<{ type: string; fn: EventListener; thirdEventParam?: boolean }>;
+}
+
+//@ts-ignore
+HTMLDivElement.prototype.realAddEventListener = HTMLDivElement.prototype.addEventListener;
```

Comment:
Maybe you could extend the type and create a new one with the added function so that you don't have to ignore it in the line above